package.json

```diff
@@ -10,8 +10,11 @@
   "dependencies": {
     "bootstrap": "^5.1.3",
     "core-js": "^3.6.5",
+    "json-editor-vue3": "^1.0.4",
```

Comment:
lets only keep the dependency we used in the end (not both) or we will get confused about which we are actually using. you can manually remove the one that you did not use in the end